app/views/comments/create.js.erb

```diff
@@ -0,0 +1,11 @@
+console.log("<%= @comment.body %>")
+
+var added_comment = $("<%= escape_javascript(render 'comments/comment', comment: @comment) %>");
```

Comment:
`escape_javascript` has an alias of `j`, so you could use that instead to save typing.

Also, since `render 'comments/comment', comment: @comment` is all conventionally named— you can replace it with simply: `render: @comment` if you want.

Altogether, you could have:

```suggestion
var added_comment = $("<%= j(render @comment) %>");
```